packages/containers/src/AppLoader/AppLoader.md

```diff
@@ -0,0 +1,45 @@
+# App Loader
+
+This component is here to make http calls before the first render of the application. During the loading, it show the application's loader.
+
+## Where i put it ?
+
+On your top level component, you can place it this way :
+
+```html
+<AppLoader>
+    <div>
```

Comment:
is that div needed ?

Reply:
to make it work with react 15, i think this is required, no ?

Reply:
So I think the div must be in the AppLoader container.
So we can inject before and after children and that will let us remove our custom App components in every projects

Reply:
Hmm I'm not sure to understand the `to make it work with react 15`.  Without the div, that makes all sub components the children. So in your AppLoader container, you use `{children}` in your JSX to render them, that's fine (and already the case).

(Don't forget to remove this div in the doc)